sympy/solvers/solveset.py

```diff
@@ -455,6 +455,12 @@ def solveset_real(f, symbol):
     original_eq = f
     f = together(f)
 
+    # in this un-like in solveset_complex expression should only
```

Comment:
--> In this, unlike in solveset_complex, expression should only

It's a minor nitpick, but I had a hard time reading it.